gridfs/__init__.py

```diff
@@ -48,14 +48,17 @@ def __init__(self, database, collection="fs", disable_md5=False):
         :Parameters:
           - `database`: database to use
           - `collection` (optional): root collection to use
-          - `disable_md5` (optional): When True, MD5 checksums will not be
-            computed for uploaded files. Useful in environments where MD5
-            cannot be used for regulatory or other reasons. Defaults to False.
+
+        .. versionchanged:: 4.0
+           Removed the `disable_md5` parameter.
```

Comment:
Also mention that gridfs no longer generates an md5sum

Reply:
Updated to link to the migration guide which describes the change in more detail.